README.md

```diff
@@ -113,6 +113,7 @@ Awesome DeFi apps you can deploy on Akash
 - [Ethereum 2.0](Ethereum_2.0)
 - [POKT Network](pokt_network)
 - [Bitcoin Cash Node](bitcoincashnode)
+- [Zcash Node](zcashnode)
```

Comment:
Please rename "zcashnode" to "Zcash" or other way around so it matches the directory you are adding ("Zcash" directory)